service/src/test/java/com/ge/predix/acs/service/policy/evaluation/PolicyEvaluationWithAttributeConnectorTest.java

```diff
@@ -24,8 +24,8 @@
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.ge.predix.acs.PolicyContextResolver;
-import com.ge.predix.acs.attribute.connectors.ResourceAttributeConnector;
-import com.ge.predix.acs.attribute.connectors.SubjectAttributeConnector;
+import com.ge.predix.acs.attribute.readers.ExternalResourceAttributeReader;
+import com.ge.predix.acs.attribute.readers.ExternalSubjectAttributeReader;
```

Comment:
As discussed, we're sticking to the `Reader` suffix for connector-related internal abstractions and the `Connector` suffix for external abstractions